src/environment.js

```diff
@@ -34,6 +34,12 @@ const ENV_VARS = {
   IPFS_GATEWAY() {
     return process.env.REACT_APP_IPFS_GATEWAY || ''
   },
+  CUSTOM_HTTP_SUBGRAPH_ENDPOINT() {
+    return process.env.REACT_APP_CUSTOM_HTTP_SUBGRAPH_ENDPOINT || ''
+  },
+  CUSTOM_WS_SUBGRAPH_ENDPOINT() {
+    return process.env.REACT_APP_CUSTOM_WS_SUBGRAPH_ENDPOINT || ''
+  },
```

Comment:
Nitpick: I might alphabetize this file; right now it's a bit hard to find the exact key I'm looking for.

Also curious—should we prefer providing the `endpoint()` defaults here instead of the `local-settings`? That might also allow us to simplify / align the env variable names to just `SUBGRAPH_HTTP_ENDPOINT` and `SUBGRAPH_WS_ENDPOINT`.